### PR TITLE
ColdFusion Variable Naming

### PR DESCRIPTION
**Pull request by grayhaze**
These changes cover adding some extra information on variable naming best practises for CF, plus a small reshuffle of the sections to group things together logically.

### CODE REVIEW

**Comment by simonbingham** [comment]
Do we need to add a rule about `attributes` scoped variables being lowercase? Or have I just made that rule up? :)

**Comment by grayhaze** [comment]
Can you give an example @simonbingham?

**Comment by simonbingham** [comment]
Slack discussion: https://globaldev.slack.com/archives/coldfusion/p1426154514000390

:+1: :tractor:

**Comment by takinmade** [comment]
:+1:

**Comment by geethanair** [comment]
:+1: :dart:

File: languages/coldfusion.md [hunk 1]
```diff
@@ -21,18 +21,21 @@
 - Constant names should be in `UPPERCASE_WITH_UNDERSCORES`
 - Don't use variable prefixes (e.g. [Hungarian Notation][hungarian]) - instead
   just name the variables sensibly
-- Scopes should be in lowercase: `form.firstName`
-- All built-in ColdFusion functions should be in `lowercase()`
-- Custom functions and component names should be `lowerCamelCased()`
-- Built-in operators should be in uppercase: `EQ`, `NEQ`
-- Boolean values should be uppercase: `TRUE` and `FALSE`
+- Variable names should be descriptive without being too verbose: `memberId` not `id`
```
Comment (timblair) [inline languages/coldfusion.md]: Please ensure that all lines are wrapped at 80 characters.

**Comment by timblair** [comment]
@simonbingham: `attributes` is effectively a meta-scope, in that it's not an "official" scope. As such, I don't think we should make reference to it in this document.

**Comment by simonbingham** [comment]
Agreed. Thanks Tim.

**Comment by timblair** [comment]
Thanks for the updates, @grayhaze :zap:

**Pull request merged**